docs/source/intervention_models/insecticide_treated_nets/index.rst

```diff
@@ -0,0 +1,241 @@
+Insecticide Treated Bed Nets for Malaria Prevention
+====================================================
+
+.. contents::
+   :local:
+   :depth: 2
+
+Malaria is a major cause of preventable mortality in areas endemic to the disease such as Sub-Saharan Africa. Malaria is a mosquito-born disease and prevention of malaria transmission can include vector control measures (such as indoor residual spraying of insecticides and insecticide-treated bed nets) as well as drug-based prevention therapies (such as :ref:`intermittent prevention therapy during pregnancy <maternal_malaria_prevention_therapy>`). Vulnerable groups to malaria disease include children under five as well as pregnant women (malaria infection during pregnancy can lead to poor health outcomes in infants).
+
+This intervention document focuses on insecicide treated bed nets for malaria prevention.
+
+.. list-table:: Abbreviations
+  :widths: 15 15 15
+  :header-rows: 1
+
+  * - Abbreviation
+    - Definition
+    - Note
+  * - ITN
+    - Insecticide treated nets
+    - 
+  * - ANC
+    - Antenatal care
+    - 
+
+Intervention Overview
+-----------------------
+
+Insecticide treated bed nets effectively reduce malaria incidence (and therefore its associated maladies) through the prevention/reduction of mosiquto bites. Children under five and pregnant women are identified as priority groups for the ITN intervention due to their vulnerability to malaria. Due to their high effectiveness and low cost, ITNs have been freely distributed through governmental/other campaigns, as discussed in [Belay-and-Deressa-2008]. However, despite attempts to distribute ITNs, access to the intervention remains limited in some area and a gap between ITN access and use remains [Deressa-et-al-2011]. ITN use during pregnancy has been associated with factors such as household wealth [Deressa-et-al-2014], education [Yitayew-et-al-2018], and ANC attendance [Ouedraogo-et-al-2019].
+
+.. todo::
+
+  Add greater discussion of successes and failures of ITN intervention programs
+
+.. list-table:: Affected Outcomes
+  :header-rows: 1
+
+  * - Outcome
+    - Effect
+    - Modeled?
+    - Note 
+  * - Birthweight
+    - Increase in population mean value
+    - Yes
+    - Effect modified by parity in [Gamble-et-al-2007]_
+  * - Preterm birth
+    - Protective relative risk
+    - No
+    - Statistically insigificant from [Gamble-et-al-2007]_
+  * - Maternal hemoglobin
+    - Increase in population mean value
+    - No
+    - Statistically insignificant from [Gamble-et-al-2007]_
+
+Baseline Coverage Data
+++++++++++++++++++++++++
+
+On September 1st, 2021 we performed a literature search on PubMed using the search terms "insecticide treated nets ethiopia pregnan*", which yielded 21 results. The results were screened and relevant studies are summarized in the below. (References are not yet linked on this page but can be found here: :code:`J:\Project\simulation_science\ciff_malnutrition\literature\itn_coverage`)
```

Comment:
This is something that has been extensively modeled by IHME and Malaria Atlas Project (MAP) teams, and we should use the values from the most recent MAP estimates, which are probably available as a GBD covariate

Reply:
Thanks for pointing this out. I have reached out to a couple of people and will make an update once I hear back from the modelers!